src/pages/Patients/VerifyPatient.tsx

```
@@ -85,7 +89,14 @@ export default function VerifyPatient(props: { facilityId: string }) {
       });
     }
   }, [phone_number, year_of_birth, partial_id, verifyPatient]);
-
+  if ((props.facilityId && isLoading) || (!patientData && !isError)) {
+    return (
+      <div className="space-y-4">
+        <CardListSkeleton count={1} />
+        <CardGridSkeleton count={4} />;
+      </div>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)